### PR TITLE
✨ [amp story player] Up Next

### LINKED ISSUE #29635: [amp-story-player] Up next
The up-next feature should show a prompt at the end of the story. It should include information about the next story and a timer.

### PR DESCRIPTION
**Pull request by connielei**
- Add up next feature that will trigger on a `storyEnd` event.
- Load in Poppins font.

Notes: 
- Removed `signalNavigation` from `go` into `show`.
- Have to add the story side that will let player know user is on last page of a story and have the player receive that message and dispatch a `storyEnd` event or extract functionality from `initializeUpNext` into a function that will trigger once the player receives the message.
- Haven't added tests yet because the trigger functionality isn't finalized yet.

Demo: https://cool-project2020.web.app/

Closes #29635

/cc @Enriqe @newmuis @bettyashley @proyectoramirez

### CODE REVIEW

File: src/amp-story-player/amp-story-player-impl.js [hunk 1]
```diff
@@ -393,6 +431,144 @@ export class AmpStoryPlayer {
...
+   * @param {!Element} textContainer
+   * @return {Object<string, Element>}
+   * @private
+   */
+  createUpNextTexts_(textContainer) {
+    textContainer.classList.add('story-player-up-next-text-container');
+
+    const text = this.doc_.createElement('p');
+    text.classList.add('story-player-up-next-text');
+    text.textContent = 'UP NEXT in 10';
```
Comment (Enriqe) [inline src/amp-story-player/amp-story-player-impl.js:555]: We'll need to localize these strings. I wonder if we can use the `LocalizationService` from `amp-story` for this.

https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story-localization-service.js#L28

File: src/amp-story-player/amp-story-player-impl.js [hunk 2]
```diff
@@ -393,6 +431,144 @@ export class AmpStoryPlayer {
...
+   * @private
+   */
+  initializeUpNext_() {
+    if (this.stories_.length === 1) {
+      return;
+    }
+
+    const {container, card, text, title, author} = this.createUpNextElement_();
+
+    this.element_.addEventListener('storyEnd', () => {
```
Comment (Enriqe) [inline src/amp-story-player/amp-story-player-impl.js:449]: We'll need to send this as a post message from the story, and register that handler in the player.

File: src/amp-story-player/amp-story-player-impl.js [hunk 3]
```diff
@@ -393,6 +431,144 @@ export class AmpStoryPlayer {
...
+   * @param {Element} text
+   * @private
+   */
+  startUpNext_(container, text) {
+    let counter = 10;
+    text.textContent = 'UP NEXT in ' + counter;
+    container.classList.remove('story-player-hide');
+
+    this.interval_ = setInterval(
+      function (next) {
```
Comment (Enriqe) [inline src/amp-story-player/amp-story-player-impl.js:488]: we can use the newer ES syntax here:

(next) => {
  counter--; 
  ...
}

**Comment by ghelapratik** [comment]
@connielei - The demo at https://cool-project2020.web.app/ is not working.

**Pull request closed without merging**